contributed_definitions/nyaml/NXcomponent_em.yaml

```diff
@@ -3,35 +3,28 @@ doc: |
   Base class for components used in an electron microscope.
   
   The electron microscope can be a real one or a simulated microscope.
-  The key motivation behind this generalization is the observation that in all
-  cases a controlled electron beam is generated in reality or that beam is simulated
-  and this beam is then used or modified in a controlled manner for the purpose
-  of studying physical interaction mechanisms of the beam with matter.
-  Here it does not matter whether one considers a real specimen or a simulated one.
-  
-  Using a common description for the real experiment in the lab and - what is
-  typically a simplification of it - via a computer simulation, has the benefit
-  that many pieces of information can be stored in the same way. In effect,
-  users are guided with finding information and unnecessary descriptive
-  variety for what are exactly the same concept is avoided to work towards
-  more interoperability.
-  
-  Another motivation to make no fundamental distinction between a scanning and
-  a transmission electron microscope is that both are electron microscopes whose
-  components are often very similar.
-# `point Electronic GmbH <https://www.pointelectronic.de>`_
 type: group
 NXcomponent_em(NXobject):
```

Comment:
Agree on this point. NXcomponent_em I proposed with exactly this idea in mind and I am happy that you see this also as a useful approach for other techniques. I propose then to implement the relevant changes though in a separate issue and PR also because currently virtually all changes on NXem shouldnt have an effect on the functionality of the parser, but changing NXcomponent_em to NXcomponent will have.

Reply:
Ok, sounds good. Let me know when you start this because then we can think about which other classes to base off of `NXcomponent `